console/src/App/App.js

```diff
@@ -159,7 +159,6 @@ const AppBase = kind({
 						{title: 'Apps', icon: 'apps'}
 					]}
 					onSelect={onSelect}
-					selected={index}
```

Comment:
Since `index` prop already has `index` passed down to it (L211/L212), it'd be redundant and I'd say we should reserve `selected` as a boolean type prop.